src/react/NewsroomWithdraw.tsx

```diff
@@ -0,0 +1,174 @@
+  userAccount?: EthAddress;
+  multisigBalance?: number;
+}
+
+export class NewsroomWithdraw extends React.Component<NewsroomWithdrawProps, NewsroomWithdrawState> {
+  public static contextType: React.Context<ICivilContext> = CivilContext;
+  public context!: React.ContextType<typeof CivilContext>;
+
+  public constructor(props: NewsroomWithdrawProps) {
+    super(props);
+    this.state = {};
+  }
+
+  public async componentDidMount(): Promise<void> {
+    const userAccount = await this.context.civil!.accountStream.first().toPromise();
+    const multisigAddress = await this.props.newsroom.getMultisigAddress();
+
+    if (!multisigAddress) {
+      // This can only happen if user created contract manually.
+      alert(
+        `This newsroom is owned by ${multisigAddress}, which is not a multisig wallet. You'll have to withdraw using another method.`,
+      );
+      return;
+    }
+
+    this.setState({
+      // @TODO Would be nice if this auto updated
+      multisigBalance: await this.context.civil!.accountBalance(multisigAddress),
+      userAccount,
+    });
+  }
+
+  public render(): JSX.Element {
+    return (
+      <Wrapper>
+        <Copy>
+          <Heading>Withdraw funds</Heading>
+          <p>
+            Transfer or withdraw funds from your Newsroom Wallet to collect proceeds from Boosts. You’ll be able to
+            exchange ETH for fiat currency. Reminder: only Newsroom Officers can access the Newsroom Wallet.
+          </p>
+          <p>
+            <a href="#TODO">Learn More &gt;</a>
+          </p>
+        </Copy>
+        <BalanceAndButton>
+          <p>
+            Newsroom balance:{" "}
+            <Query query={ethPriceQuery}>
+              {({ loading, error, data }) => {
+                if (loading || typeof this.state.multisigBalance === "undefined") {
+                  return <LoadingIndicator />;
+                }
+                return <b>${(data.storefrontEthPrice * this.state.multisigBalance).toFixed(2)}</b>;
+              }}
+            </Query>
+            <br />
+            {typeof this.state.multisigBalance !== "undefined" && <>({this.state.multisigBalance.toFixed(4)} ETH)</>}
+          </p>
+          <TransactionButton
+            Button={props => (
+              <BoostButton disabled={props.disabled} onClick={props.onClick}>
+                Withdraw
+              </BoostButton>
+            )}
+            disabled={!this.state.multisigBalance || !this.state.userAccount}
+            transactions={[
+              {
+                transaction: this.withdrawTx,
+                postTransaction: this.postTransaction,
+              },
+            ]}
+          >
+            Withdraw
+          </TransactionButton>
+        </BalanceAndButton>
+      </Wrapper>
+    );
+  }
+
+  private withdrawTx = async (): Promise<TwoStepEthTransaction<any> | void> => {
+    if (!this.state.multisigBalance || !this.state.userAccount || !this.context.civil || !(window as any).ethereum) {
```

Comment:
we should try to limit where we use `(window as any).ethereum` since this is going to go away once we support multiple wallets (ie, portis doesn't have an injected web3 provider). if possible we could try to contain this to either a function within the civil context, or perhaps a helper function in this repo so the logic is only in one place

Reply:
my preference would be to just add a TODO for this. we already have to fix this in multiple places during login improvements, would prefer not to delay this for that refactor

Reply:
Sounds good, will add a todo, and we should grep for all references to this when we make those updates.